src/app/atoms/CopyButton.tsx

```diff
@@ -96,11 +98,12 @@ const CopyButton: FC<CopyButtonProps> = ({
         }
         {...rest}
         onClick={handleCopyPress}
+        {...setTestID(ExploreSelectors.valueButton)}
       >
         {children}
       </button>
 
-      <input ref={fieldRef} value={text} readOnly className="sr-only" />
+      <input ref={fieldRef} value={text} readOnly className="sr-only" {...setTestID(ExploreSelectors.valueInput)} />
```

Comment:
the [same](https://github.com/madfish-solutions/templewallet-extension/pull/852/files#r1093600182)